.ci/install-esy.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [[ $ESY_BUILD == YES ]]; then
+  npm --global install esy@0.3.x
+  esy --version
+else
+  echo "Nothing to be done."
```

Comment:
Is this `else` branch needed? Seeing this message in the log is not informative IMO.

Reply:
It looked strange to see the script marked as ran but _no output whatsoever_. Perhaps I can change the text to `"Skipping esy installation."` ?

Reply:
Ended up joining the `install` scripts after removing the `OCAML` env variable from the esy build. This is no longer an issue.